changelogs/unreleased/1821-prydonius

```diff
@@ -0,0 +1 @@
+report backup progress in PodVolumeBackups and expose progress in the velero backup describe --details command
```

Comment:
Should we additionally mention the bump in restic version?

Reply:
Good shout, will do that

Reply:
updated, ptal!